content/api/resources/operations/object/liquidity-pool-withdraw.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Liquidity Pool Deposit Object
```

Comment:
c/p, this title should be for _Withdrawal_

Reply:
nice catch, thanks!